camera_sensors: 
  - name: 'kinect'
    hfov: 58
    vfov: 45.0
    image_width: 640
    image_height: 480
  - name: 'left_camera'
    hfov: 60
    vfov: 33.75
    image_width: 1280
    image_height: 720 
  - name: 'right_camera'
    hfov: 60
    vfov: 33.75
    image_width: 1280
    image_height: 720 
